#!/usr/bin/env python
PACKAGE = "perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("mask_mode", int_t, 0,               "0 for trapezoid mask mode, 1 for standard mask, 2 for modified mask path", 1, 0, 2)

exit(gen.generate(PACKAGE, "perception", "PreprocessingMaskMode"))
